### ISSUE TITLE
[Proposal]Able to add the localization to all matched localization class.

### ISSUE THREAD

**Issue by andy840119** [ppy/osu-localisation-analyser#54]
Another request beside #53 is able to select and add the localization string to those matched localization class.
![image](https://user-images.githubusercontent.com/9100368/172082980-3c023fa1-41b3-43a1-87bb-b672b4c92cc9.png)
Will have more option like Like `Insert the localization to the Chat/AAAStrings.cs` and `Insert the localization to the Chat/BBBStrings.cs`

The proposal is figured out while implementing this issue.
https://github.com/karaoke-dev/karaoke/pull/1365

It should be able merge into just localization class called `GeneratorStrings` and `DetectorStrings`.
It will let the translation easy to be added if user can selected exist matched localization classes in the `Localisation`.

**Comment by smoogipoo** [comment]
I don't agree with adding more options. Two is the max I'd want to display, any more than that will result in mental overload. Even two options is already feeling like too much.

**Comment by andy840119** [comment]
Okay, i think the only remain option to collect the localization is following the discussion in here:
https://github.com/karaoke-dev/karaoke/issues/1265#issuecomment-1148280807

Or created to the `CommonStrings.cs`, then move it by hand.
